.github/workflows/tests.yml

```diff
@@ -33,7 +33,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        airflow-version: ["2.1.4", "2.2.5", "2.3.0"]
+        include:
+          - python-version: "3.6"
+            airflow-version: "2.2.5"
```

Comment:
Could you please tell in brief, what does it mean?

Reply:
I've checked it. All ok, you can use the following code reproduce behaviour

`docker run --platform linux/amd64 -v $(pwd)/airflow-clickhouse-plugin:/usr/share/code -it python:3.10-slim-buster bash`

<img width="1077" alt="Screenshot 2022-05-24 at 19 41 32" src="https://user-images.githubusercontent.com/1254188/170088286-3b8d1989-7d4a-418f-82a9-bfe8b50fb954.png">

`docker run --platform linux/amd64 -v $(pwd)/airflow-clickhouse-plugin:/usr/share/code -it python:3.6-slim-buster bash`
<img width="1400" alt="Screenshot 2022-05-24 at 19 40 46" src="https://user-images.githubusercontent.com/1254188/170088298-ff34616e-f103-4750-b969-aa4c76ed5179.png">